src/extension.ts

```diff
@@ -24,8 +24,10 @@ export let outputChannelDep: any;
 export function activate(context: vscode.ExtensionContext) {
   let disposableFullStack = vscode.commands.registerCommand(
     Commands.TRIGGER_FULL_STACK_ANALYSIS,
-    (uri: vscode.Uri) =>
-      multimanifestmodule.dependencyAnalyticsReportFlow(context, uri)
+    (uri: vscode.Uri) => {
+      let fileUri = uri ? uri : vscode.window.activeTextEditor.document.uri;

```

Comment:
nit:
1. Can we use `const` here instead of `let`?
2. Good to have a comment about why `uri` is `null/undefined`.

Reply:
@arajkumar done. ready for review again